src/builder-context.c

```diff
@@ -192,6 +196,7 @@ builder_context_constructed (GObject *object)
   self->cache_dir = g_file_get_child (self->state_dir, "cache");
   self->checksums_dir = g_file_get_child (self->state_dir, "checksums");
   self->ccache_dir = g_file_get_child (self->state_dir, "ccache");
+  self->sdk_defaults = g_file_get_child (self->state_dir, "sdk-defaults.json");
```

Comment:
Hmm, this is not right. The state dir can be shared by all kinds of builds.

Reply:
The problem is that a build might overwrite this file. So we do not want to take it from the build directory. Do you have a better way to read a file from the SDK?

Reply:
Would making it a directory and store them by project name work?